client/components/common/ManageRequiredReportsDisclosure/index.jsx

```diff
@@ -0,0 +1,294 @@
+              phase: `IS_${actionData.phase}`
+            }
+          });
+
+          const updatedReport = data.requiredReport[`${action}RequiredReport`];
+
+          if (updatedReport) {
+            updateAtBrowserCombinations(updatedReport, action);
+          }
+
+          await triggerUpdate();
+        }, successMessage);
+      } catch (error) {
+        showErrorModal(
+          `Error ${
+            action.charAt(0).toUpperCase() + action.slice(1)
+          }ing Required Report`,
+          error.message
+        );
+      }
+    },
+    [triggerLoad, triggerUpdate, updateAtBrowserCombinations, showErrorModal]
+  );
+
+  const handleCreateRequiredReport = useCallback(
+    createData =>
+      handleRequiredReportAction(
+        'create',
+        createData,
+        createRequiredReport,
+        'Adding Phase requirement to the required reports table'
+      ),
+    [handleRequiredReportAction, createRequiredReport]
+  );
+
+  const handleDeleteReqReport = useCallback(
+    deletionData => {
+      handleRequiredReportAction(
+        'delete',
+        deletionData,
+        deleteRequiredReport,
+        'Deleting Phase requirement from the required reports table'
+      ),
+        handleCloseDeleteModal();
+    },
+    [handleRequiredReportAction, deleteRequiredReport, handleCloseDeleteModal]
+  );
+
+  const handleUpdateReport = useCallback(
+    updateData => {
+      handleRequiredReportAction(
+        'update',
+        updateData,
+        updateRequiredReport,
+        'Updating Phase requirement in the required reports table'
+      ),
+        handleCloseUpdateModal();
+    },
+    [handleRequiredReportAction, updateRequiredReport, handleCloseUpdateModal]
+  );
+
+  const handleCloseUpdateModal = () => {
+    setPendingUpdateReport(null);
+  };
+
+  const handleCloseDeleteModal = () => {
+    setPendingDeleteReport(null);
+  };
+
+  return (
+    <>
+      <DisclosureContainer data-testid="manage-required-reports-disclosure">
+        <span>Add required reports for a specific AT and Browser pair</span>
+        <CreateRequiredReportForm
+          ats={ats}
+          handleCreate={handleCreateRequiredReport}
+        />
+        <ThemeTableHeaderH2 style={{ padding: '1rem' }}>
+          Required Reports
+        </ThemeTableHeaderH2>
+        <ThemeTable bordered responsive>
+          <thead>
+            <tr>
+              <th>Phase</th>
+              <th>AT</th>
+              <th>Browser</th>
+              <th>Edit</th>
+            </tr>
+          </thead>
+          <tbody>
+            {atBrowserCombinations?.map(({ at, browser, phase }) => {
+              return (
+                <tr key={`${at.id}-${browser.id}-${phase}`}>
+                  <td>
+                    <PhasePill fullWidth={false} forHeader={true}>
+                      {phase}
+                    </PhasePill>
+                  </td>
+                  <td>{at.name}</td>
+                  <td>{browser.name}</td>
```

Comment:
`browser.name` isn't coming through for the rows on the Data Management page (which also affects the edit and delete functions on that page)

![screenshot showing the browser name is missing for all rows](https://github.com/user-attachments/assets/8653a5db-85a8-4694-b6fc-6f71462c3d92)

Reply:
Oh interesting. I wasn't testing on the Data Management page but realizing now that this disclosure wasn't intended to exist on that page and a boolean prop in `ManageTestQueue` was being used to exclude it previously. I'll do that.